components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.html

```diff
@@ -44,6 +44,13 @@ <h2 slot="title">Create {{ objectNoun | titlecase }}</h2>
             <chef-button tertiary (click)="modifyID = true" id="edit-button-object-modal">Edit ID</chef-button>
           </chef-toolbar>
         </div>
+        <div *ngIf="showProjectsDropdown">
+          <app-projects-dropdown
+            [projects]="projectsAssignable"
```

Comment:
[nit] s/projectsAssignable/assignableProjects (sounds a bit more natural)

Reply:
🤖🤖🤖🤖🤖I'll change that in my follow up PR so I don't have to wait for CI again!

Reply:
Fixed!